ios/Sample/sampleTests/sampleTests.swift

```diff
@@ -31,4 +31,16 @@ class sampleTests: XCTestCase {
         assert(contentHash == "5f1da1a2d0feb614dd03d71e87928b8e449ac87614479332aced3a701f916743")
     }
     
+    func testHmac256Signature() {
```

Comment:
This is the swift test double for https://github.com/novoda/aws-v4-signer-java/pull/6/files#diff-7b91a47366e48d7e31257e95ae72045d